crates/aptos-telemetry-service/src/clients/mod.rs

```diff
@@ -38,14 +38,18 @@ pub mod victoria_metrics_api {
             raw_metrics_body: Bytes,
             extra_labels: Vec<String>,
             encoding: String,
+            timestamp: usize,
         ) -> Result<reqwest::Response, anyhow::Error> {
             let labels: Vec<(String, String)> = extra_labels
                 .iter()
                 .map(|label| ("extra_label".into(), label.into()))
                 .collect();
 
             self.inner
-                .post(format!("{}api/v1/import/prometheus", self.base_url))
+                .post(format!(
+                    "{}api/v1/import/prometheus?timestamp={}",
+                    self.base_url, timestamp
+                ))
```

Comment:
The timestamp should be part of the `query()` in line 55. We already send some extra labels in  query params, and timestamp should be part of that.